lambdas/service/app.py

```diff
@@ -741,6 +775,7 @@ def fetch_dss_files(uuid):
 
 def _dss_files(uuid, fetch=True):
     query_params = app.current_request.query_params
+    validate_params(query_params, True, fileName=str, wait=int, requestIndex=int)
     url = config.dss_endpoint + '/files/' + urllib.parse.quote(uuid, safe='')
     file_name = query_params.pop('fileName', None)
     wait = query_params.pop('wait', None)
```

Comment:
Also, this is not your change, but it looks like https://github.com/DataBiosphere/azul/pull/1135/commits/efa0c70195a4bf89f62acee3dce6bbeb9a0cc4f2#diff-a3dec4f6b35a08e7cb29086cb8766c3aR793 and the following line do nothing and can be removed.